components/containers/DynamicForm/DynamicForm.tsx

```diff
@@ -5,21 +5,17 @@ import Head from "next/head";
 import { Form } from "../../forms/Form/Form";
 import { TextPage } from "../../forms/TextPage/TextPage";
 import { getProperty, getRenderedForm } from "../../../lib/formBuilder";
-import { PublicFormSchemaProperties } from "../../../lib/types";
+import { DynamicFormProps } from "../../../lib/types";
 import { useRouter } from "next/router";
 import { useFlag } from "../../../lib/hooks/useFlag";
-
-interface DynamicFormProps {
```

Comment:
Weird that the interface was a duplicate of the one in `types.ts`. Nice catch :)